esrs-macros/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use proc_macro::TokenStream;
+
+use proc_macro2::Ident;
+use quote::quote;
+use syn::DeriveInput;
+
+#[proc_macro_derive(Event)]
+/// Implements [`Debug`] for a struct or enum, with certain fields redacted.
+///
+/// See the [crate level documentation](index.html) for flags and modifiers.
+pub fn derive_event(item: TokenStream) -> TokenStream {
```

Comment:
Do you think that is worth it to have this? 

Pros: it's easy to use, it's just a `derive(esrs::Event)`; if we implement the event versioning this could be definitively worth it to add a versioning behaviour

Cons: breaking change; need to publish the `esrs-macros` crate;

Reply:
Yes, that was exactly my point. Ok, maybe i will put it into when there will be a real need. Thank you!